doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,166 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
+     - ❌
+     - ✓ [6]_
+     - ✔️
+     - ❌
+     - ✔️
+   * - sphere
+     - ❌
+     - ✔️
+     - ✔️
+     - ❌
+     - ✔️
+   * - arcball
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+     - ❌
+
+
+.. [1] The way it was prior to v3.10; this is also MATLAB's style
+.. [2] Mouse controls roll too (not only azimuth and elevation)
+.. [3] Figure reacts the same way to mouse movements, regardless of orientation (no difference between 'poles' and 'equator')
+.. [4] Returning mouse to original position returns figure to original orientation (rotation is independent of the details of the path the mouse took)
+.. [5] The style has a corresponding natural implementation as a mechanical device
+.. [6] While it is possible to control roll with the ``trackball`` style, this is not immediately obvious (it requires moving the mouse in large circles) and a bit counterintuitive (the resulting roll is in the opposite direction)
+
+You can try out one of the various mouse rotation styles using:
+
+.. code::
+
+    import matplotlib as mpl
+    mpl.rcParams['axes3d.mouserotationstyle'] = 'trackball'  # 'azel', 'trackball', 'sphere', or 'arcball'
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from matplotlib import cm
+
+    ax = plt.figure().add_subplot(projection='3d')
+
+    X = np.arange(-5, 5, 0.25)
+    Y = np.arange(-5, 5, 0.25)
+    X, Y = np.meshgrid(X, Y)
+    R = np.sqrt(X**2 + Y**2)
+    Z = np.sin(R)
+
+    surf = ax.plot_surface(X, Y, Z, cmap=cm.coolwarm,
+                           linewidth=0, antialiased=False)
+
+    plt.show()
+
+Alternatively, create a file ``matplotlibrc``, with contents::
+
+    axes3d.mouserotationstyle: trackball
+
+(or any of the other styles, instead of ``trackball``), and then run any of
+the :ref:`mplot3d-examples-index` examples.
+
+The size of the virtual trackball, sphere, or arcball can be adjusted
+by setting :rc:`axes3d.trackballsize`. This specifies how much
+mouse motion is needed to obtain a given rotation angle (when near the center),
+and it controls where the edge of the sphere or arcball is (how far from
+the center, hence how close to the plot edge).
+The size is specified in units of the Axes bounding box,
+i.e., to make the arcball span the whole bounding box, set it to 1.
+A size of about 2/3 appears to work reasonably well; this is the default.
+
+Both arcballs (``mouserotationstyle: sphere`` and
+``mouserotationstyle: arcball``) have a noticeable edge; the edge can be made
+less abrupt by specifying a border width, :rc:`axes3d.trackballborder`.
+This works somewhat like Gavin Bell's arcball, which was
+originally written for OpenGL [Bell1988]_, and is used in Blender and Meshlab.
+Bell's arcball extends the arcball's spherical control surface with a hyperbola;
+the two are smoothly joined. However, the hyperbola extends all the way beyond
+the edge of the plot. In the mplot3d sphere and arcball style, the border extends
+to a radius `trackballsize/2 + trackballborder`.
```

Comment:
Small docs build error to fix here:

`/home/circleci/project/doc/api/toolkits/mplot3d/view_angles.rst:172: WARNING: py:obj reference target not found: trackballsize/2 + trackballborder [ref.obj]`

Reply:
Fixed it; but some tests on macos are still failing... what to do about it?

Reply:
The macos checks can be ignored, they're failing on main right now and unrelated to the PR. However I think the mypy error might be real here.

Reply:
> [...] However I think the mypy error might be real here.

But where? The output reads:
```
error: not checking stubs due to mypy build errors:
lib/matplotlib/tests/test_basic.py:14: error: "type: ignore" comment without error code  [ignore-without-code]
lib/matplotlib/tests/test_cbook.py:189: error: "type: ignore" comment without error code (consider "type: ignore[assignment]" instead)  [ignore-without-code]
lib/matplotlib/tests/test_pickle.py:20: error: "type: ignore" comment without error code  [ignore-without-code]
lib/matplotlib/tests/test_legend.py:405: error: "type: ignore" comment without error code  [ignore-without-code]
lib/matplotlib/tests/test_colors.py:1418: error: "type: ignore" comment without error code (consider "type: ignore[explicit-override, misc]" instead)  [ignore-without-code]
lib/matplotlib/tests/test_collections.py:459: error: "type: ignore" comment without error code  [ignore-without-code]
lib/matplotlib/tests/test_backend_gtk3.py:10: error: "type: ignore" comment without error code  [ignore-without-code]
lib/matplotlib/tests/test_axes.py:36: error: "type: ignore" comment without error code  [ignore-without-code]
lib/matplotlib/tests/test_axes.py:2979: error: If condition is always true  [redundant-expr]
lib/matplotlib/tests/test_backend_qt.py:18: error: "type: ignore" comment without error code (consider "type: ignore[attr-defined]" instead)  [ignore-without-code]
lib/matplotlib/tests/test_backends_interactive.py:89: error: "type: ignore" comment without error code  [ignore-without-code]
stubtest: exit 1 (22.65 seconds) /home/runner/work/matplotlib/matplotlib> python tools/stubtest.py pid=2843
.pkg: _exit> python /opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pyproject_api/_backend.py True mesonpy
  stubtest: FAIL code 1 (107.61=setup[84.97]+cmd[22.65] seconds)
  evaluation failed :( (109.34 seconds)
Error: Process completed with exit code 1.
```

I thought I'm only involved via `test_axes3d.py` (not mentioned in the error report) ...?

Reply:
`mypy` gives me:
```
C:\Users\Mischa\AppData\Local\Programs\Python\Python312\Lib\site-packages\PySide6\QtGui.pyi:1094: error: unexpected indent  [syntax]
Found 1 error in 1 file (errors prevented further checking)
```
both with the original or modified view_angles.rst, doesn't matter...
Is it that I'm too far behind the main branch by now?

Reply:
It looks like this is failing in other CI so not a part of this change! My mistake, thanks for taking a look.